src/factory-in-proc.js

```diff
@@ -34,6 +37,21 @@ class FactoryInProc {
     this.exec = options.exec
   }
 
+  /**
+   * Utility method to get a temporary directory
+   * useful in browsers to be able to generate temp
+   * repos manually
+   *
+   * *Here for completeness*
+   *
+   * @param {string} type - the type of the node
+   * @param {function(Error, string)} callback
+   * @returns {undefined}
+   */
+  tmpDir (type, callback) {
+    callback(null, tmpDir(type === 'js'))
+  }
+
```

Comment:
@dryajov you are assuming that a Node in Proc only exists in Node.js.. but it can also be browser.